src/test/java/com/univocity/parsers/csv/CsvFormatTest.java

```diff
@@ -0,0 +1,296 @@
+    Reflector.setField(objectUnderTest, "comment", '\u0000');
+    Reflector.setField(objectUnderTest, "lineSeparator", null);
+    Reflector.setField(objectUnderTest, "normalizedNewline", '\u0000');
+    objectUnderTest.setQuote('\u8001');
+    objectUnderTest.setDelimiter('\u0000');
+    Reflector.setField(objectUnderTest, "charToEscapeQuoteEscaping", null);
+    objectUnderTest.setQuoteEscape('\u0001');
+    Reflector.setField(objectUnderTest, "lineSeparatorString", null);
+    final char ch = '\u0000';
+
+    // Act
+    final boolean retval = objectUnderTest.isCharToEscapeQuoteEscaping(ch);
+
+    // Assert result
+    Assert.assertEquals(false, retval);
+  }
+
+  /*
+   * Test generated by Diffblue Deeptest.
+   * This test case covers:
+   *  - conditional line 190 branch to line 190
+   */
+
+  @Test
+  public void isCharToEscapeQuoteEscapingInputNotNullOutputTrue() throws InvocationTargetException {
+
+    // Arrange
+    final CsvFormat objectUnderTest =
+        (CsvFormat)Reflector.getInstance("com.univocity.parsers.csv.CsvFormat");
+    Reflector.setField(objectUnderTest, "comment", '\u0000');
+    Reflector.setField(objectUnderTest, "lineSeparator", null);
+    Reflector.setField(objectUnderTest, "normalizedNewline", '\u0000');
+    objectUnderTest.setQuote('\u8001');
+    objectUnderTest.setDelimiter('\u0000');
+    Reflector.setField(objectUnderTest, "charToEscapeQuoteEscaping", null);
+    objectUnderTest.setQuoteEscape('\u0001');
+    Reflector.setField(objectUnderTest, "lineSeparatorString", null);
+    final char ch = '\u0001';
+
+    // Act
+    final boolean retval = objectUnderTest.isCharToEscapeQuoteEscaping(ch);
+
+    // Assert result
+    Assert.assertEquals(true, retval);
+  }
+
+  /*
+   * Test generated by Diffblue Deeptest.
+   * This test case covers:
+   *  - conditional line 129 branch to line 129
+   */
+
+  @Test
+  public void isDelimiterInputNotNullOutputFalse() throws InvocationTargetException {
+
+    // Arrange
+    final CsvFormat objectUnderTest =
+        (CsvFormat)Reflector.getInstance("com.univocity.parsers.csv.CsvFormat");
+    Reflector.setField(objectUnderTest, "comment", '\u0000');
+    Reflector.setField(objectUnderTest, "lineSeparator", null);
+    Reflector.setField(objectUnderTest, "normalizedNewline", '\u0000');
+    objectUnderTest.setQuote('\u0000');
+    objectUnderTest.setDelimiter('\u0001');
+    Reflector.setField(objectUnderTest, "charToEscapeQuoteEscaping", null);
+    objectUnderTest.setQuoteEscape('\u0000');
+    Reflector.setField(objectUnderTest, "lineSeparatorString", null);
+    final char ch = '\u0000';
+
+    // Act
+    final boolean retval = objectUnderTest.isDelimiter(ch);
+
+    // Assert result
+    Assert.assertEquals(false, retval);
+  }
+
+  /*
+   * Test generated by Diffblue Deeptest.
+   * This test case covers:
+   *  - conditional line 129 branch to line 129
+   */
+
+  @Test
+  public void isDelimiterInputNotNullOutputTrue() throws InvocationTargetException {
```

Comment:
I like this - I wonder if they know if the delimiter and the quote field are the same, this method will return like this?

Reply:
:female_detective: